e2e-tests/specs/all.e2e.js

```diff
@@ -307,34 +346,74 @@ describe('Hide CTA(s):', async () => {
     });
 });
 
-describe('Carousel', async () => {
-    const cloneConfig = structuredClone(config);
-    cloneConfig.collection.layout.container = 'carousel';
-
-    const state = btoa(JSON.stringify(cloneConfig));
-    const url = `${serverPath}/html/e2e/carousel.html?state=${state}`;
-
-    it('Carousel Title should exist', async () => {
+// contains good examples of clicking behaviors
+describe('Filter and Search Behaviors', async () => {
+    it('MWPW-137132: can filter results using both search terms and filters by url', async () => {
+        const cloneConfig = structuredClone(config);
+        const state = btoa(JSON.stringify(cloneConfig));
+        const url = `${serverPath}/html/e2e/e2e-grid.html?state=${state}&&ch_Topic=Creativity%2520and%2520Design`;
         await browser.url(url);
         await browser.setTimeout({ script: 50000 });
-        const title = await $('.consonant-CarouselInfo-collectionTitle').getText();
-        await expect(title).toEqual('Lorem Ipsum');
+        const cardRes = await $('.consonant-LoadMore-text').getText();
+        expect(cardRes).toEqual('Showing 3 of 3 cards');
     });
-
-    it('All cards should appear', async () => {
+    it('MWPW-136333: can use "clear all" to clear terms from url and grid', async () => {
+        const cloneConfig = structuredClone(config);
+        const state = btoa(JSON.stringify(cloneConfig));
+        const url = `${serverPath}/html/e2e/e2e-grid.html?state=${state}&sh_=Ze&ch_Topic=Stock%252CPersonalization`;
         await browser.url(url);
         await browser.setTimeout({ script: 50000 });
-        const grid = await $('.consonant-CardsGrid');
-        await expect(grid).toHaveChildren(8);
+        const clearAll = await $('.consonant-LeftFilters-clearLink');
+        await clearAll.click();
+        const cardRes = await $('.consonant-LoadMore-text').getText();
+        expect(cardRes).toEqual('Showing 5 of 8 cards');
```

Comment:
@sheridansunier Why showing 5 of 8 cards is the correct outcome? if you clear all filters then all 8 should be showing .. unless search filter is not removed.

Reply:
b/c of pagination